src/test/java/com/wikia/webdriver/common/core/api/ForumBoardContent.java

```diff
@@ -0,0 +1,46 @@
+package com.wikia.webdriver.common.core.api;
+
+import java.io.IOException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.IOException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)